notebooks/224-stable-diffusion-text-to-image/README.md

```diff
@@ -0,0 +1,33 @@
+# Text-to-image generation with Stable Diffusion and OpenVINO
```

Comment:
"... and OpenVINO" in the and of the title looks excessive considering that this is all about OpenVINO. You can add "and OpenVINO native API" if you want to highlight implementation details or remove it at all.

Reply:
removed